data/sheragi/archaeology missions.txt

```diff
@@ -112,18 +112,24 @@ mission "Deep Archaeology 3"
 		personality waiting
 		government "Deep Security"
 		ship "Raven (Afterburner)" "D.P.S. Wyrd"
```

Comment:
While this PR is up, I want to ask whether the `D.P.S. Wyrd`, Skuld, and Verthandi should have different prefixes than other Deep Security ships, `D.S.S.`, and have a government called `Deep` instead of something like `Deep Security (Hostile)`? MZ added these missions, so we can't just ask him...

Reply:
I think they're supposed to be deep intelligence, not deep security.

Reply:
Then it should be more clear. What does `D.P.S.` stand for? Deep Police Ship? `D.S.S.` is Deep Security Ship, of course. So why not `D.I.S.`, Deep Intelligence Ship?

Reply:
The weird thing is that they **are** listed as "Deep Security" in Deep Archaeology 3. I think that either it should be listed as "Deep" both times (with the reputation being set to -1000 in Deep Archaeology 5), or it should be listed as "Deep Security" using a separate government in Deep Archaeology 5. But yes, I think this should be a separate PR.